src/index.html

```diff
@@ -2,13 +2,51 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#menu">
+          <img class="logo"
+          src="images/logo.png"
+          alt="logo">
+        </a>
+      </div>
+      <div class="menu">
+        <nav class="nav">
+          <ul class="nav__item">
+            <li class="nav__list">
+              <a href="#APPLE" class="nav__link is-active">APPLE</a>
+            </li>
+            <li class="nav__list">
```

Comment:
Add one empty line between sibling blocks inside the same parent block for both of them.

Reply:
```suggestion
            </li>
            
            <li class="nav__list">
```